seatunnel-translation/seatunnel-translation-base/src/main/java/org/apache/seatunnel/translation/source/CoordinatedSource.java

```diff
@@ -109,9 +117,11 @@ private void createSplitEnumerator() throws Exception {
                                 throw new RuntimeException(e);
                             }
                         }
+                        // The information read in the status is added to the Map
```

Comment:
Would it be better to describe this process?       
   restoredState.forEach(
                    (subtaskId, splitBytes) -> {
                        if (subtaskId == -1) {
                            return;
                        }
                        List<SplitT> restoredSplitState = new ArrayList<>(splitBytes.size());
                        for (byte[] splitByte : splitBytes) {
                            try {
                                restoredSplitState.add(splitSerializer.deserialize(splitByte));
                            } catch (IOException e) {
                                throw new RuntimeException(e);
                            }
                        }
                        // The information read in the status is added to the Map
                        restoredSplitStateMap.put(subtaskId, restoredSplitState);
                    });